CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Introduces the `conjur-config-cluster-prep.yaml` and `conjur-config-namespace-prep.yaml` raw Kubernetes manifests generated from their corresponding Helm charts. These manifests provide an alternative method of configuring a Kubernetes cluster for the deployment of Conjur-authenticated applications for users unable to use  Helm in their environment.
+  [cyberark/conjur-authn-k8s-client#338](https://github.com/cyberark/conjur-authn-k8s-client/issues/338)
+- Added user-configurable Helm values for the names of resources created by the `conjur-config-namespace-prep` Helm chart
+  [cyberark/conjur-authn-k8s-client#383](https://github.com/cyberark/conjur-authn-k8s-client/issues/383)
 
 ## [0.21.1] - 2021-09-09
```

Comment:
Do you want to remove the 0.21.1 changelog entry in this PR so your release PR is clean?

Reply:
Yeah okay let's just do it here.